plugin/editorconfig.vim

```diff
@@ -242,9 +243,12 @@ function! s:UseConfigFiles() abort " Apply config to the current buffer {{{1
     endfor
 
     if s:editorconfig_core_mode ==? 'vim_core'
-        call s:UseConfigFiles_VimCore()
+        if s:UseConfigFiles_VimCore() == 0
```

Comment:
I changed this from `is 0` to `== 0` --- my preference is to use arithmetic operators on numbers.

Reply:
Sounds good since this one's obviously a number type under your control. FWIW I default to `is` out of habit because vim has bizarre coercion rules so that for instance `'walrus' == 0` returns true.

Reply:
TIL - thanks!